src/gl/context.cpp

```diff
@@ -308,18 +326,85 @@ void Context::setScene(const SurfaceBuffer &surfaceBuffer, mat4x4 sunView, bool
   // calculate pixel area (A[i]*cos(theta) for each pixel of the surface)
   // multiplies by the number of pixels to get projected sunlit surface area
 
-  pixelArea = (right - left) * (top - bottom) / (size * size);
+  auto const pixelArea = (right - left) * (top - bottom) / (size * size);
+
+  mat4x4_ortho(projection, left, right, bottom, top, -near_, -far_);
+  mat4x4_mul(mvp, projection, view);
+
+  setMVP();
+
+  return pixelArea;
+}
+
+float Context::setScene(mat4x4 sunView, bool clipFar) {
```

Comment:
@mbadams5  we are merging this into a single `setScene()` method. Is this signature ever used anywhere?